taf/developer_tool.py

```diff
@@ -268,16 +298,20 @@ def create_repository(
         Indicates if the created repository is a test authentication repository
     """
     yubikeys = defaultdict(dict)
+    auth_repo = AuthenticationRepo(repo_path)
+    repo_path = Path(repo_path)
+
+    if not _check_if_can_create_repository(auth_repo):
```

Comment:
Could you just call `auth_repo._tuf_repository` and if it raises an error, you will know there there is no (or invalid) tuf repository at the path.

Reply:
This is meant to check if repository already exists. If the directory contains a git repository with metadata files, reinitializing metadata files could result in an invalid repo (e.g. versions of metadata being reverted to 1)